linkinator.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recurse": true,
+  "skip": [
```

Comment:
let's add a tracking bug to come back around and clean this up after the commit lands

Reply:
done :ok_hand: